scripts/core/filters/index.js

```diff
@@ -214,4 +214,8 @@ export default angular.module('superdesk.core.filters', [])
         var cleanedValue = value || '';
 
         return cleanedValue.replace('_', ' ');
-    });
+    })
+    .filter('trimHtmlTags', () => function(value) {
```

Comment:
we have something defined already, maybe it should go into some `core/utils` and be used in both
https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/authoring/helpers.js#L99

Reply:
@petrjasek okay, I'll do that. You mean creating a new `core/utils/index.js` and exporting a `stripHtmlRaw` function that I can use in the filter as well as the tests right?

Reply:
yep, maybe just make it `core/utils.js` not sure if we need more now